f/medical/fn_addPullOutAction.sqf

```diff
@@ -0,0 +1,44 @@
+#include "\a3\functions_f_mp_mark\Revive\defines.inc"
+
+if (!hasInterface) exitWith {};
+
+params ["_vehicle"];
+
+private _actionId = _vehicle getVariable ["#revPullOutAction", -1];
+
+// Vehicle already has a pull out action, don't try to add another
+if (_actionId != -1) exitWith {};
+
+// Code to run when the hold-action completes
+private _action = {
+	params ["_target"];
+	
+	{
+		private _unit = _x;
+		
+		// Pull the unit out if they are down
+		if (IS_DISABLED(_unit)) then {
+			[_unit] remoteExec ["f_fnc_pullOutUnit", _unit];
+		}
+	} forEach crew _target;
+};
+
+_actionId = [
+	_vehicle,
+	"Pull out wounded",
+	"\a3\ui_f\data\IGUI\Cfg\holdactions\holdAction_unbind_ca.paa",
+	"\a3\ui_f\data\IGUI\Cfg\holdactions\holdAction_unbind_ca.paa",
+	"(_this distance _target < 5) && {[_target] call f_fnc_vehicleHasWounded}",

```

Comment:
Note to later self: this needs a not in vehicle check. Currently getting into the vehicle with a downed player shows the actions.